crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -1426,7 +1428,7 @@ impl EntityCommands<'_> {
     /// # bevy_ecs::system::assert_is_system(my_system);
     /// ```
     #[allow(clippy::should_implement_trait)]
```

Comment:
This `allow` could have been removed when the method was renamed from `add` to `queue`.  That doesn't need to be part of this PR, but I noticed it because it was added in the PR you're reverting.

Reply:
```suggestion
```